freq/views.py

```diff
@@ -771,21 +785,28 @@ def locations(self):
 
     @property
     def timeseries_(self):
-        statistic = self.request.session['map_'].get(
-            'dropdown_0', {'value': 'mean'})['value']
+        gw_type, statistic = [
+            x.strip(' ') for x in self.request.session['map_'].get(
+                'dropdown_0', {'value': 'mean'})['value'].split('|')
+        ]
+        print('gw_type [{}], statistic [{}]'.format(gw_type, statistic))
         timeseries = GroundwaterTimeSeries()
         timeseries.queries = {
-            "name": self.request.GET.get("groundwater_type", "GWmMSL")
+            "name": gw_type
         }
         timeseries.bbox(*self.coordinates, statistic=statistic,
                         **self.time_window)
-        return timeseries.ts_to_dict(
+        result = timeseries.ts_to_dict(
             start_date=jsdt.datestring_to_js(self.request.session['map_']['datepicker'][
                 'start']),
             end_date=jsdt.datestring_to_js(self.request.session['map_'][
                                                'datepicker']['end']),
             date_time='str'
         )
+        if result["dates"]['start'] and result['dates']['end']:
```

Comment:
Nee zeker weten niet. In dat geval is de ts_to_dict methode stuk. Wel bugprone. Ik kan evt. afvangen en een error raisen.